libs/components/lookup/src/lib/modules/country-field/country-field.component.scss

```diff
@@ -1,6 +1,6 @@
 @use 'libs/components/theme/src/lib/styles/mixins' as mixins;
 @use 'libs/components/theme/src/lib/styles/variables' as *;
-@use 'node_modules/intl-tel-input/build/css/intlTelInput';
+@import 'node_modules/intl-tel-input/build/css/intlTelInput.css';
```

Comment:
Switching to `@import` to [load this file as CSS (not SASS)](https://sass-lang.com/documentation/at-rules/import/), which fixes an issue with `esbuild` and resolving the background image `url()`.